Content.Server/GameObjects/Components/Doors/AirlockComponent.cs

```diff
@@ -75,11 +74,18 @@ private bool BoltLightsVisible
             set
             {
                 _boltLightsWirePulsed = value;
-                UpdateWiresStatus();
                 UpdateBoltLightStatus();
             }
         }
 
+        private const float AUTO_CLOSE_DELAY_FAST = 1;
```

Comment:
This should probably be called `AutoCloseDelayFast` to follow C# naming conventions.